shared/src/telemetry/event.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::PathBuf;
+
+use serde_json::json;
+
+#[derive(Clone)]
+pub enum TelemetryEvent {
+    Deployment {
+        extension: String,
+    },
+    DirectoryStatus {
+        extension: String,
+        path: PathBuf,
+        size: u64,
+    },
+}
+
+impl TelemetryEvent {
+    pub fn name(&self) -> String {
+        match self {
+            Self::Deployment { extension } => format!("{extension} Deployment"),
```

Comment:
So we will still get the one-time "pg_bm25 Deployment" ping one they deploy? I would like for us to keep that, in addition to the directory status

Reply:
This is still happening, it's just done by the background worker now in pgrx/Rust.